File: Seminar5/HomeWork/Model/Complex/ComplexPrint.java
package HomeWork.Model.Complex;

public class ComplexPrint {

    Complex complex;

    public ComplexPrint(Complex complex) {
        this.complex = complex;
    }

    public StringBuilder print() {
        StringBuilder res = new StringBuilder();
        if (complex.getImage() > 0) {
            res.append(complex.getReal() + " + " + complex.getImage() + "i");
        } else if (complex.getImage() < 0) {
            res.append(complex.getReal() + "" + complex.getImage() + "i");
        } else {
            res.append(complex.getReal());
        }
        return res;

    }

}

File: FinalTaskCalculator/Model/Complex/ComplexPrint.cs
using System.Text;

namespace ModelComplex
{

    public class ComplexPrint
    {

        Complex complex;

        public ComplexPrint(Complex complex)
        {
            this.complex = complex;
        }

        public StringBuilder Print()
        {
            StringBuilder res = new StringBuilder();
            if (Convert.ToDouble(complex.Image) > 0)
            {
                res.Append(complex.Real + " + " + complex.Image + "j");
            }
            else if (Convert.ToDouble(complex.Image.ToString()) < 0)
            {
                res.Append(complex.Real + "" + complex.Image + "j");
            }
            else
            {
                res.Append(complex.Real);
            }
            return res;
        }

    }
}